test/uk/gov/hmrc/economiccrimelevyregistration/navigation/CharityRegistrationNumberPageNavigatorSpec.scala

```diff
@@ -53,7 +54,8 @@ class CharityRegistrationNumberPageDeregisterNavigatorSpec extends SpecBase {
         val otherEntityJourneyData = OtherEntityJourneyData
           .empty()
           .copy(
-            companyRegistrationNumber = Some(number)
+            companyRegistrationNumber = Some(number),
+            isCtUtrPresent = Some(random[Boolean])
```

Comment:
You are using arbitrary and forAll at the top of the test. This boolean can go there. You don't need random.

Reply:
Changed.